content/docs/tasks/telemetry/logs/collecting-logs/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Collecting Logs
+description: This task shows you how to configure Istio to collect and customize logs.
+weight: 10
+keywords: [telemetry,logs]
+---
+
+This task shows how to configure Istio to automatically gather telemetry for
+services in a mesh. At the end of this task, a new log stream will be enabled
+for calls to services within your mesh.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used
+as the example application throughout this task.
+
+## Before you begin
+
+* [Install Istio](/docs/setup/kubernetes) in your cluster and deploy an
+  application. This task assumes that Mixer is setup in a default configuration
+  (`--configDefaultNamespace=istio-system`). If you use a different
+  value, update the configuration and commands in this task to match the value.
+
+## Collecting new logs data
+
+1.  Create a new YAML file to hold configuration for the new log
+    stream that Istio will generate and collect automatically.
+
+    Save the following as `new_logs.yaml`:
+
+    {{< text yaml >}}
```

Comment:
Use {{< text yaml downloadas="new_logs.yaml" >}}. This will make the download button that shows up over the text block default to that file name.

Reply:
that's awesome!

Reply:
So, I was able to add text to get it to download as "new_logs.yaml" but it downloads as an empty file.  What have I done wrong?